reactNativeProject/src/screens/Main/components/Button/styles.js

```diff
@@ -0,0 +1,10 @@
+import { StyleSheet } from "react-native";
+
+export default StyleSheet.create({
```

Comment:
Save the `StyleSheet.create()` to a `styles` const. Move `export default` to the end of the file.